src/components/incentives/PointsIncentivesTooltipContent.tsx

```diff
@@ -0,0 +1,80 @@
+import { Trans } from '@lingui/macro';
+import { Box } from '@mui/material';
+
+import { Link } from '../primitives/Link';
+
+export const EthenaAirdropTooltipContent = ({ points }: { points: number }) => {
```

Comment:
This file can probably be optimized into one component if it continues to grow. As its all the same except for the text and the link i think right?

Reply:
Yes you're right, I'll refactor this